### ISSUE TITLE
Whether single cell data of different cell components are suitable for analysis after integration

### ISSUE THREAD

**Issue by Dolphin-dfy** [ventolab/CellphoneDB#199]
A single cell dataset has A cell, and a single cell dataset has B cells. After batch integration of the two, can cellphonedb be used to analyze the interaction between A cells and B cells? Specifically, I found two single-cell cohorts in the public database, one with epithelial cells and immune cells, and the other with immune cells only. After the two cohorts were integrated, the appropriate interaction between epithelial cells and immune cells was analyzed.

**Comment by ktroule** [comment]
Yes you can used CellPhoneDB to analyze two independent datasets. You can either join both datasets and normalize the counts or use the integration corrected counts if you think there is a reason for this. The only requirement is that counts must be non-negative.
